use-cases/rag-on-gke/backend/src/requirements.txt

```diff
@@ -0,0 +1,7 @@
+fastapi
+llama-index==0.11.21
+pydantic
+typing
+uvicorn
```

Comment:
Can you pin your versions so it is reproducible.

Reply:
fixed as well